### PR TITLE
Orders Clean Up

### PR DESCRIPTION
**Pull request by nuhuhmutebi**

### CODE REVIEW

File: configuration/concepts/orders_obstetricsGynecology.csv [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+f16675de-d6bb-4fd1-914c-a25d8cf59465,,,OBG1009 - Césarienne (suivi a la clinique),OBG1009 - Césarienne (suivi a la clinique),Césarienne (suivi a la clinique),Césarienne (suivi a la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+c2510453-994d-4a49-9016-a77e883382c0,,,OBG1010 - Césarienne (suivi hors de la clinique),OBG1010 - Césarienne (suivi hors de la clinique),Césarienne (suivi hors de la clinique),Césarienne (suivi hors de la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+3517d312-8a22-4f60-a098-e9cc0bb5a49b,,,OBG1011 - Césarienne + Hysterectomie (suivi a la clinique),OBG1011 - Césarienne + Hysterectomie (suivi a la clinique),Césarienne + Hysterectomie (suivi a la clinique),Césarienne + Hysterectomie (suivi a la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+865bb6bf-b38b-4a6d-a013-834e450e26d5,,,OBG1012 - Césarienne + Hysterectomie (suivi hors la clinique),OBG1012 - Césarienne + Hysterectomie (suivi hors la clinique),Césarienne + Hysterectomie (suivi hors la clinique),Césarienne + Hysterectomie (suivi hors la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
```
Comment (ultra-mine) [inline configuration/concepts/orders_obstetricsGynecology.csv:13]: @nuhuhmutebi I seem to still see accents **Césarienne** which should be **Cesarienne**

File: configuration/concepts/orders_obstetricsGynecology.csv [hunk 2]
```diff
@@ -23,13 +23,13 @@ d15a58e0-83a2-4d98-8eb7-3a902e2c524b,,,OBG1021 - Curetage thérapeutique/Avortem
...
-681d5790-8069-45c6-96ce-9ebf2c254471,,,OBG1026 - Délivrance Spontanée( accouchement hors HSC),OBG1026 - Délivrance Spontanée( accouchement hors HSC),Délivrance Spontanée( accouchement hors HSC),Délivrance Spontanée( accouchement hors HSC),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+f7bd36ae-ce11-4668-b69f-015f960824ac,,,OBG1025 - Délivrance Artificielle (accouchement hors clinique),OBG1025 - Délivrance Artificielle (accouchement hors clinique),Délivrance Artificielle (accouchement hors clinique),Délivrance Artificielle (accouchement hors clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+681d5790-8069-45c6-96ce-9ebf2c254471,,,OBG1026 - Délivrance Spontanée( accouchement hors clinique),OBG1026 - Délivrance Spontanée( accouchement hors clinique),Délivrance Spontanée( accouchement hors clinique),Délivrance Spontanée( accouchement hors clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
```
Comment (ultra-mine) [inline configuration/concepts/orders_obstetricsGynecology.csv:27]: Same here

**Review by ultra-mine — CHANGES_REQUESTED** [review]

File: configuration/concepts/orders_obstetricsGynecology.csv [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-865bb6bf-b38b-4a6d-a013-834e450e26d5,,,OBG1012 - Césarienne + Hysterectomie (suivi hors l'HSC),OBG1012 - Césarienne + Hysterectomie (suivi hors l'HSC),Césarienne + Hysterectomie (suivi hors l'HSC),Césarienne + Hysterectomie (suivi hors l'HSC),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+f16675de-d6bb-4fd1-914c-a25d8cf59465,,,OBG1009 - Césarienne (suivi a la clinique),OBG1009 - Césarienne (suivi a la clinique),Césarienne (suivi a la clinique),Césarienne (suivi a la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+c2510453-994d-4a49-9016-a77e883382c0,,,OBG1010 - Césarienne (suivi hors de la clinique),OBG1010 - Césarienne (suivi hors de la clinique),Césarienne (suivi hors de la clinique),Césarienne (suivi hors de la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+3517d312-8a22-4f60-a098-e9cc0bb5a49b,,,OBG1011 - Césarienne + Hysterectomie (suivi a la clinique),OBG1011 - Césarienne + Hysterectomie (suivi a la clinique),Césarienne + Hysterectomie (suivi a la clinique),Césarienne + Hysterectomie (suivi a la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/orders_obstetricsGynecology.csv]: We change this to **Cesarienne** rather than **Césarienne**

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/orders_obstetricsGynecology.csv (hunk 3 above)
Comment (nuhuhmutebi) [reply configuration/concepts/orders_obstetricsGynecology.csv]: Done

Comment (nuhuhmutebi) [reply configuration/concepts/orders_obstetricsGynecology.csv]: Please consider the forced push

**Pull request closed without merging**